src/discordtipbot/discordbot.cr

```diff
@@ -460,14 +460,13 @@ class DiscordBot
       node = @tip.node_balance
       return if node.nil?
       unclaimed = node - @tip.db_balance
-      reply(msg, "Unclaimed coins: **#{unclaimed}** #{@config.coinname_short}")
+      return reply(msg, "Unclaimed coins: **#{unclaimed}** #{@config.coinname_short}")
     end
 
-    if cmd.size == 3
-      if cmd[1] == "balance"
-        bal = @tip.get_balance(cmd[2].to_u64)
-        reply(msg, "**#{cmd[2]}**'s balance is: **#{bal}** #{@config.coinname_short}")
-      end
+    if cmd[1]? == "balance"
+      return reply(msg, "**ERROR**: You fergot to supply a ID to check balance of")
```

Comment:
The next line is never reached because this line always returns unconditionally. I think you mean to do something like `unless cmd[2]?`